src/test/java/de/retest/web/SimplePageIT.java

```diff
@@ -31,6 +31,8 @@ void simple_html_page_should_be_checked( final WebDriver driver ) throws Excepti
 		final Path simplePagePath = Paths.get( "src/test/resources/pages/simple-page.html" );
 		driver.get( simplePagePath.toUri().toURL().toString() );
 
+		Thread.sleep( 1000 );
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS2925))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=84&issues=AWahP8E1rChX7_AlIBKF&open=AWahP8E1rChX7_AlIBKF)

[ISSUE_KEY]: AWahP8E1rChX7_AlIBKF